.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,4 +11,4 @@ Please explain the changes you made here:
 
 - A description of the problem you're trying to solve
 - An overview of the suggested solution
-- If the feature changes current behavior, reasons why your solution is better
\ No newline at end of file
+- If the feature changes current behavior, reasons why your solution is better
```

Comment:
Not sure how this made it in?

Reply:
That's weird, I can add `.github` to the `.gitignore`

Reply:
hmm it may be easier to make a fresh branch from develop, apply the same changes and submit a new pull request to the develop branch

Reply:
I switched the target from `master` to `develop` and I was going to rebase it. But it will probably be cleaner to wait for push access from Josh and create a new branch on the official repo.

Reply:
Sounds good

Reply:
@lastmjs you're good to go now. Pull `develop`, branch from it, push back here, and then request a PR from your branch to `develop`. 🙏